### PR TITLE
db.rake & test.rake should not do code outside task context

### PR DESCRIPTION
**Pull request by esparta**
There is code that is executed regardless the task you are mean to execute, for examples, if you try to do rubocop, it will always try to connect... 
This also will get some improvement on how much time it takes to run a task:

``` sh
# before
$ time rake rubocop
Running RuboCop...
Inspecting 212 files
....................................................................................................................................................................................................................

212 files inspected, no offenses detected

real    0m10.431s
user    0m3.100s
sys     0m6.356s

# after
$time rake rubocop
Running RuboCop...
Inspecting 212 files
....................................................................................................................................................................................................................

212 files inspected, no offenses detected

real    0m4.797s
user    0m2.072s
sys     0m2.452s
```

### CODE REVIEW

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -89,6 +92,12 @@ end
   end
 
   def config
+    require 'db/connection'
```
Comment (bernardoamc) [inline lib/tasks/db.rake]: Maybe I'm misunderstanding something, but can we move this `require` to the top and just call `DB::Config.new` in the appropriate methods?

File: lib/tasks/db.rake [hunk 2]
```diff
@@ -89,6 +92,12 @@ end
...
     DB::Config.new
   end
+
+  def connect
```
Comment (bernardoamc) [inline lib/tasks/db.rake]: Same here in this method.

**Comment by bernardoamc** [comment]
Thanks for the PR @esparta ! <3

**Comment by esparta** [comment]
Done the required changes, ·bernardoamc.
Thanks for your time.

File: lib/tasks/db.rake (hunk 1 above)
Comment (kytrinyx) [inline lib/tasks/db.rake]: It's safe, but I do find that loading unnecessary code slows things down, and I think that on principle we should avoid it.

**Comment by kytrinyx** [comment]
Thanks so much for taking the time to fix this!

**Pull request merged**